deepgram/transcription.py

```diff
@@ -65,8 +68,22 @@ async def __call__(self) -> 'LiveTranscription':
     async def _start(self) -> None:
         asyncio.create_task(self._receiver())
         self._ping_handlers(LiveTranscriptionEvent.OPEN, self)
+
+        connection_retries = 0
+
         while not self.done:
-            incoming, body = await self._queue.get()
+            try:
+                incoming, body = await asyncio.wait_for(self._queue.get(), self.MESSAGE_TIMEOUT)
+            except asyncio.TimeoutError:
+                print(f"Server didn't receive or send message within {self.MESSAGE_TIMEOUT} seconds")
+
```

Comment:
I'm not certain if we want to have the library itself printing things that could end up mixing unintentionally with user output.
```suggestion
```